app/src/pages/NotFoundPage.vue

```diff
@@ -22,12 +22,14 @@ const { isAuthenticated } = useAuth0();
                 }}
             </p>
             <div v-if="isAuthenticated" class="mt-10 flex items-center justify-center gap-x-6">
-                <RouterLink to="/" class="text-yellow-700 underline"> Back to home </RouterLink>
+                <RouterLink to="/" class="border-b-[1px] border-yellow-700 text-yellow-700">
```

Comment:
What's the reason for avoiding `underline`? That's one of the most basic CSS features so should be supported 😄

Reply:
Turns out it's not as well supported as I thought surprisingly: https://caniuse.com/mdn-css_properties_text-decoration-line_underline - but I'd still argue that this would be fine to use, since it's not that big a problem if it doesn't show up in some browsers (as long as your links also have a different color).